caffe2/CMakeLists.txt

```
@@ -452,11 +452,17 @@ if(USE_LITE_INTERPRETER_PROFILER)
   append_filelist("libtorch_edge_profiler_sources " LITE_PROFILER_SRCS)
 endif()
 
+list(APPEND LITE_AOTI_SRCS "")
+if(USE_LITE_AOTI)
+  append_filelist("inductor_core_resources" LITE_AOTI_SRCS)
+endif()
+
 # Switch between the full jit interpreter and lite interpreter
 if(BUILD_LITE_INTERPRETER)
   append_filelist("libtorch_lite_cmake_sources" LIBTORCH_CMAKE_SRCS)
   list(APPEND LIBTORCH_CMAKE_SRCS ${LITE_EAGER_SYMOBLICATION_SRCS})
   list(APPEND LIBTORCH_CMAKE_SRCS ${LITE_PROFILER_SRCS})
+  list(APPEND LIBTORCH_CMAKE_SRCS ${LITE_AOTI_SRCS})
```

Why not do something like that?
```suggestion
  if(USE_LITE_AOTI)
    append_filelist("inductor_core_resources" LIBTORCH_CMAKE_SRCS)
  endif()

    list(APPEND LIBTORCH_CMAKE_SRCS ${LITE_AOTI_SRCS})
```